### PR TITLE
feat: added git star button

### LINKED ISSUE #768: [Feature] Add GitHub stars in Navbar
### Description

Add github stars badge in the navbar to show all the github stars of linkshub

Checkout [this](https://www.minimamente.com/project/magic/) site for reference

### Screenshots

_No response_

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Bhanuteja01**
## Fixes Issue
Closes #768 

## Changes proposed
Added GitHub Star Button

## Screenshots
Screenshot (9)

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
Looks great @Bhanuteja01! :) I like the color on it

File: components/SocialMedia/SocialMediaIconsList.tsx [hunk 1]
```diff
@@ -1,16 +1,19 @@
-import { FC } from "react";
-import { IconContext } from "react-icons";
-import { FaDiscord, FaGithub, FaTwitter } from "react-icons/fa";
-import { GitHubForkButton } from "components/ForkButton/GitHubForkButton";
-
+import { FC } from 'react'
+import { IconContext } from 'react-icons'
+import { FaDiscord, FaGithub, FaTwitter } from 'react-icons/fa'
+import { GitHubForkButton } from 'components/ForkButton/GitHubForkButton'
```
Comment (krishstwt) [inline components/SocialMedia/SocialMediaIconsList.tsx:4]: Everything looks great, but why was the code style of the document changed? Shouldn't that be a separate change/pr?

**Review by krishstwt — CHANGES_REQUESTED** [review]

**Comment by Bhanuteja01** [comment]
> Looks great @Bhanuteja01! :) I like the color on it

thanks for appreciation

**Review by rupali-codes — COMMENTED** [review]
Thanks, but the color doesn't really match with LinksHub theme 👀

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Bhanuteja01. Your branch has merge conflicts, so I suggest solving them.

**Comment by Bhanuteja01** [comment]
> Thanks, but the color doesn't really match with LinksHub theme eyes

@rupali-codes  Should i change the colour or
Create issue to change the colour

**Comment by rupali-codes** [comment]
> > Thanks, but the color doesn't really match with LinksHub theme eyes
> 
> @rupali-codes Should i change the colour or Create issue to change the colour

No, you don't have create new issue.

File: components/StarButton/GitHubStarButton.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  return (
+    <Link
+      href={`https://github.com/${repo}/stargazers`}
+      passHref
+      target="_blank"
+      rel="noopener noreferrer"
+      aria-label={`Star ${repo} on GitHub`}
+    >
+      <div className="inline-flex items-center px-4 py-1 text-sm font-semibold bg-transparent text-yellow-400 border border-yellow-400 border-transparent rounded-sm transition-colors shadow-md">
```
Comment (rupali-codes) [inline components/StarButton/GitHubStarButton.tsx:33]:
```suggestion
      <div className="inline-flex items-center px-4 py-1 text-sm bg-violet-500 text-white border border-dashed border-transparent rounded-sm transition-colors shadow-md hover:bg-transparent hover:border-violet-400 hover:text-violet-400">
```

Normal:
image

on Hover: 
image

**Review by ujjawaltyagii — APPROVED** [review]
Now It looks better !!

**Review by CBID2 — APPROVED** [review]
LGTM!

**Pull request merged**